docs/user/spark/jupyter.rst

```diff
@@ -119,13 +119,122 @@ GeoMesa version ``VERSION``:
 You may also consider adding ``geomesa-tools-2.11-$VERSION-data.jar`` to include prepackaged converters for
 publicly available data sources (as described in :ref:`prepackaged_converters`),
 ``geomesa-jupyter-leaflet-2.11-$VERSION.jar`` to include an interface for the `Leaflet`_ spatial visualization
-library, and/or ``geomesa-jupyter-vegas-2.11-$VERSION.jar`` to use the `Vegas`_ data plotting library
-(see :ref:`jupyter_vegas` below).
+library (see :ref:`jupyter_leaflet`, next), and/or ``geomesa-jupyter-vegas-2.11-$VERSION.jar`` to use the `Vegas`_ data
+plotting library (see :ref:`jupyter_vegas`, below).
+
+.. _jupyter_leaflet:
+
+Leaflet for Visualization
+-------------------------
+
+The following sample notebook shows how you can use Leaflet for data visualization:
+
+
+.. code-block:: scala
+
+   classpath.addRepository("http://download.osgeo.org/webdav/geotools")
+   classpath.addRepository("http://central.maven.org/maven2")
+   classpath.addRepository("https://repo.locationtech.org/content/repositories/geomesa-releases")
+   classpath.addRepository("file:///home/username/.m2/repository")
+   classpath.add("com.vividsolutions" % "jts" % "1.13")
```

Comment:
Hmmm?  Seems like this block should be updated to jts-core version 1.14.0 and geomesa-* 2.0.*.

Reply:
see, readme's are out of date ;)
i'll fix